tests/test_validate.py

```diff
@@ -0,0 +1,42 @@
+import re
+
+from augur import validate
+
+import pytest
+
+
+class TestValidate:
+    def test_auspice_config_v2_valid(self):
+        # Implied assertion that no exceptions are raised
+        validate.auspice_config_v2(
+            "tests/data/validation/auspice_config_v2__valid.json"
+        )
+
+    def test_auspice_config_v2_invalid(self):
+        with pytest.raises(
+            validate.JsonValidationError, match="'key' is a required property"
+        ):
+            validate.auspice_config_v2(
+                "tests/data/validation/auspice_config_v2__invalid.json"
+            )
+
+    def test_export_v2_valid(self):
+        # Implied assertion that no exceptions are raised
+        validate.export_v2("tests/data/validation/v2_zika__valid.json")
+
+    def test_export_v2_invalid(self):
+        try:
+            validate.export_v2("tests/data/validation/v2_zika__invalid.json")
+        except validate.JsonValidationError as e:
+            for i, prop in enumerate(["nuc", "panels", "updated", "version"]):
```

Comment:
I have mixed thoughts about tying the tests of the validation code too closely to the schemas. The code (and tests) should function regardless of any changes to the schema. But `validate.py` does have hard-coded references to some schema files, which means the behavior of the class _does_ change when the schemas change.

I went for the easy solution as a first pass and just used real files.